stabox-backend/ormconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "type": "mysql",
+    "host": "localhost",
+    "port": 3306,
+    "username": "root",
```

Comment:
Add a file with informations about the environmental requirements. 
Like: "MySQL username root, no passwd on localhost:3306, you can change them here: *reference*"